connectors/migrations/20240129_cleanup_orphaned_notion_cache.ts

```diff
@@ -0,0 +1,20 @@
+import { sequelize_conn } from "@connectors/lib/models";
+
+async function main() {
+  await sequelize_conn.query(`
+        DELETE FROM notion_connector_page_cache_entries WHERE "connectorId" IS NULL;
```

Comment:
the connector id would be null then? i would have thought it would be a connector id set, but the connector entry does not exist anymore?

Reply:
This is not possible with a foreign key on a relational DB. There are several possible scenarios, but the default one in Sequelize (and sadly the one we have a lot in the codebase) is to have nullable foreign keys with a SET NULL when the parent resource is deleted. 

Other options include CASCADE (child resource gets deleted when parent is deleted) or RESTRICT (deleting a parent isn't possible unless all children have been deleted first)